packages/produce-promise/src/producePromise.js

```diff
@@ -4,8 +4,6 @@
 // eslint-disable-next-line spaced-comment
```

Comment:
Maybe kill this whole module first, before we invest any more effort maintaining it?

Reply:
New issue #1471